strapi/strapi
# Bump umzug to 3.1.1

### What does it do?

I updated the version of umzug and fixed the breaking changes

### Why is it needed?

To ease future development on the migration feature

### How to test it?

Create a migration file in `database/migrations` and check that it is correctly executed after starting the app.

### Related issue(s)/PR(s)

none
